src/main/java/org/jenkinsci/plugins/ghprb/GhprbRepository.java

```diff
@@ -144,16 +144,12 @@ public void createCommitStatus(AbstractBuild<?, ?> build, String sha1, GHCommitS
             } else {
                 ghRepository.createCommitStatus(sha1, state, url, message);
             }
-        } catch (FileNotFoundException ex) {
-            newMessage = "FileNotFoundException means that the credentials Jenkins is using is probably wrong. Or that something is really wrong with github.";
-            if (stream != null) {
-                stream.println(newMessage);
-                ex.printStackTrace(stream);
+        } catch (IOException ex) {
+            if (ex instanceof FileNotFoundException) {
+              newMessage = "FileNotFoundException means that the credentials Jenkins is using is probably wrong. Or the user account does not have write access to the repo.";
```

Comment:
Well the catch handling doesn't fall through in java, so the fallback capability to use comments instead of commit status was not happening when the api started throwing FNF exceptions instead of generic IO exceptions.

So this lets the fallback behavior happen in more cases.  It also updates the error message to indicate that the user account probably does not have write access to the repo, as indicated by the comments in #232